### ISSUE TITLE
fresh-server-event-handlers should now allow colocated (_islands) directory

### ISSUE THREAD

**Issue by junichi-chiba** [denoland/deno_lint#1190]
### Lint Name

fresh-server-event-handlers

### Code Snippet

Fresh 1.4, located at /routes/mypage/(_islands)/MyIsland.tsx

```ts
                  <input
                    type="checkbox"
                    checked={!myvalue}
                    onClick={() => {
                      myaction();
                    }}
                  />

```

### Expected Result

No lint errors

### Actual Result

Following lint message is displayed:

`Server components cannot install client side event handlers.`
`hint: Remove this property or turn the enclosing component into an island`

### Additional Info

Fresh 1.4 now allows placing any islands component under routes, in `(_islands)` sub-directory.
See https://deno.com/blog/fresh-1.4

### Version

deno 1.36.1 (release, aarch64-apple-darwin)
v8 11.6.189.12
typescript 5.1.6
Fresh 1.4.2

**Comment by not-ivy** [comment]
this should be fixed in #1195 however I dont think it is released yet

**Comment by lucacasonato** [comment]
This is released now.
